about.html

```diff
@@ -19,27 +20,27 @@
     </ul>
   </header>  
   <div class="nav-container">
```

Comment:
- [ ] You have done great so far, your menu looks great, but as stated in the project requirement, you are meant to replicate the given design as much as you can. Currently, you have a white space above the menu which is not in the given design, I'll suggest you remove this, :+1: :+1: 

- - Screenshot:
![Selection_078](https://github.com/NalianAchei2020/Capstone/assets/107632780/6c71351d-7dd4-46fb-97ed-d48e55dd8f68)